tsconfig.json

```diff
@@ -8,9 +8,11 @@
     "moduleResolution": "node",
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
+    "allowJs": true,
```

Comment:
why is this needed ?

Reply:
This was needed in the situation where molgenis-i18n-js.em.js was imported from a ts file; at least from the jest transpilation.